notfound/extension.py

```diff
@@ -85,11 +89,18 @@ def pathto(otheruri, resource=False, baseuri=None):
         .. note::
 
             If ``otheruri`` is a external ``resource`` it does not modify it.
+            If ``otheruri`` is a static file on Read the Docs it does not modify it.
         """
+        READTHEDOCS = os.environ.get('READTHEDOCS', False) == 'True'
+
         if resource and '://' in otheruri:
             # allow non-local resources given by scheme
             return otheruri
 
+        if READTHEDOCS and otheruri.startswith('/_/'):
```

Comment:
Do we need to special case RTD here, or should we just not have this run on any URL that is already absolute? I don't fully understand the issue, but generally absolute URL's won't need to change?

Reply:
This is a good question. I thought we have already thought about this but I didn't find a test or discussion on the issues.

I think it should be fine to not modify absolute URLs, but I'd prefer to not introduce it right now just in case I'm missing something. So, I created an issue to track, discuss and think more about this before moving forward: https://github.com/readthedocs/sphinx-notfound-page/issues/206